test/apis/v1alpha1/api_validation_integration_test.go

```diff
@@ -133,13 +131,13 @@ var _ = Describe("Test networking v1alpha1 API validation", func() {
 		})
 
 		It("should allow creating API with valid name starting with alphabet character", func() {
-			var name = "abc-123456789"
+			var nameValid = "abc-123456789"
```

Comment:
Please create the global variable nameValidStartingWithAphabet = "abc-123456789", so that we can distinguish them and reuse for all the resources.